tests/wrappers/core_wrappers_test.py

```diff
@@ -18,9 +19,30 @@
 pytest_plugins = ["tests.pytest_plugins.llvm"]
 
 
+class ObservationDummyWrapper(ObservationWrapper):
```

Comment:
Why `ObservationDummyWrapper` and not `ObservationWrapper` (using an `import .. as ..` to avoid shadowing the `ObservationWrapper` in gym module)

Reply:
Done.